imap_processing/cdf/config/imap_mag_l1a_variable_attrs.yaml

```diff
@@ -0,0 +1,157 @@
+default_attrs: &default
```

Comment:
That is in the global attribute file, since it should be the same across the board for everyone. It's accessed via the same instance of a class as these variables, so they end up together in code for easy access.